Framework/Beamline/test/ComponentInfoTest.h

```diff
@@ -946,12 +1005,12 @@ class ComponentInfoTest : public CxxTest::TestSuite {
     b.setPosition(b.root(), pos2);
     std::pair<int64_t, int64_t> interval1(0, 1);
     std::pair<int64_t, int64_t> interval2(1, 2);
-    a.setScanInterval(a.root(), interval1);
-    b.setScanInterval(b.root(), interval2);
+    functor(a, a.root(), interval1);
+    functor(b, b.root(), interval2);
```

Comment:
`functor`? It is completely unclear what this does.

Reply:
Just a function that can be injected to apply different `setScanInterval` member function calls. Between many of the tests (sync/async) this is the only difference. However, in light of the async removal (comment above), I'll get rid of the template parameter.